crates/pop-cli/src/commands/mod.rs

```diff
@@ -33,15 +37,18 @@ pub(crate) enum Command {
 	#[clap(alias = "t")]
 	#[cfg(feature = "contract")]
 	Test(test::TestArgs),
-	/// Set up the environment for development by installing required packages.
-	#[clap(alias = "i")]
-	Install(install::InstallArgs),
+	/// Remove generated/cached artifacts.
+	#[clap(alias = "C")]
+	#[cfg(feature = "parachain")]
```

Comment:
It now affect contracts too. For e2e tests, we keep the `contracts-node` binary in cache as well.
https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-contracts/src/utils/contracts_node.rs#L46